src/common/pkcs11/pkcs11.cpp

```diff
@@ -238,17 +238,15 @@ Array<uint8_t> ConvertToAttributeValue(T& val)
 
 Error GenPIN(String& pin)
 {
-    if (pin.MaxSize() == 0) {
-        return ErrorEnum::eNone;
-    }
+    const auto PinLength = Min<size_t>(cGenPINLen, pin.MaxSize());
```

Comment:
Shouldn't it be a lower camel case variable?

Reply:
For variable yes, for local const values probably we should use upper camel.

Reply:
we use prefix `c` for const: `cPinLength`

Reply:
fixed